src/classes/RD2_EntryFormController.cls

```diff
@@ -114,4 +156,18 @@ public with sharing class RD2_EntryFormController {
         }
         private set;
     }
+
+    /**
+     * @description Builld a map of field permissions to return the the LWC page to assign with rendering custom picklist fields
```

Comment:
This is confusing to me.  Would it be correct to reword as "Builld a map of field permissions for use by LWC so it can validate access permission without a return trip to server."